### ISSUE TITLE
Does not work on Magento v1.9.2.4

### ISSUE THREAD

**Issue by thomasklosinsky** [quafzi/magento-checkout-newsletter-subscription#1]
Don't get any newsletter subscription button in checkout...
no log entries... :(

**Comment by quafzi** [comment]
Did you find any reasons? It should appear next to agreements.

**Comment by thomasklosinsky** [comment]
Not yet... not log entries so far.
Any blocks blocked by restrictions would appear in there, but nothing...
Any chance to check on your side?
Since it's not a theme related option it should appear either way, right?

**Comment by Webmasterei** [comment]
Works in my 1.9.3.0

**Comment by amcguireweb** [comment]
Doesn't work in my 1.9.3.0 - does it work with Magento's persistant one page checkout?

**Comment by quafzi** [comment]
@amcguireweb That might be a good hint!

@thomasklosinsky @Webmasterei Do you use the persistant feature?

**Comment by amcguireweb** [comment]
I got it. I may have missed this somewhere, but your first comment here tipped me off...

System > Configuration > Sales > Checkout > Checkout Options > Enable Terms and Conditions must be enabled.

**Comment by thomasklosinsky** [comment]
Yeah, I might have used the persistent feature. I already got it running with another module but will definitely check on other store setups in future and will let you know!

**Comment by FiloDaCo** [comment]
Is there a way to get this extension capturing subscriber's first and last name and working with Magento's subscription confirmation process (System > Configuration > Customers/Newsletter) ?
